src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/vedtak/AvsnittUtil.kt

```diff
@@ -108,10 +109,16 @@ internal object AvsnittUtil {
         val totalrente = vedtaksbrevsdata.perioder.sumOf { it.resultat.rentebeløp }
         val totalForeldetBeløp = vedtaksbrevsdata.perioder.sumOf { it.resultat.foreldetBeløp ?: BigDecimal.ZERO }
         val totalTilbakekrevesBeløpUtenSkattMedRenter = vedtaksbrevsdata.perioder.sumOf { it.resultat.tilbakekrevesBeløpUtenSkattMedRenter }
+        val totalFeilutbetaltBeløp = vedtaksbrevsdata.perioder.sumOf { it.kravgrunnlag.feilutbetaltBeløp }
+        val sammenslåttPeriodeKravgrunnlag =
+            HbKravgrunnlag(
+                riktigBeløp = førstePeriode.kravgrunnlag.riktigBeløp,
+                utbetaltBeløp = førstePeriode.kravgrunnlag.utbetaltBeløp,
```

Comment:
Se igjennom konsekvensen av om disse beløpene er for første periode vs totalFeilutbetaltBeløp

Reply:
`riktigBeløp` og `utbetaltBeløp` brukes kun i tester og kan kanskje fjernes for å hindre forvirring senere?

Reply:
Laget en oppgave på dette i Favro